CONTRIBUTING.md

```diff
@@ -176,4 +176,7 @@ Note that it isn't possible to easily run a single unit test from Xcode. Instead
 
 The [Podspec](./RealmJS.podspec) file follows [the CocoaPods Podspec syntax](https://guides.cocoapods.org/syntax/podspec.html). It describes how Realm JS is build for React Native apps on iOS and is automatically discovered by the React Native CLI when running `pod install` from a users project.
 
-The Podspec file expects to be located in the `./node_modules/realm/` folder of a React Native project with a `./ios/Podfile` file. The easiest way to verify that changes to the Podspec is not breaking installing the library, install Realm JS in a new React Native app, make the changes to the `./node_modules/realm/RealmJS.podspec` and run `pod lib lint --verbose` from the `./node_modules/realm/` directory. Consider adding `--no-clean` to prevent the CocoaPods CLI from deleting the temporary project created during linting. Once linting passes, try installing the pod and running the consuming app on an iOS device.
+The Podspec file expects to be located in the `./node_modules/realm/` folder of a React Native project with a `./ios/Podfile` file. The easiest way to verify that changes to the Podspec is not breaking installing the library, install Realm JS in a new React Native app, make the changes to the `./node_modules/realm/RealmJS.podspec` and build the app.
```

Comment:
Should we add something like this to the documentation at https://realm.io?

Reply:
I thought about it, but I think this will be the case for 99% of all installments. It might be a little too detailed for the vast majority.